lmsweb/views.py

```diff
@@ -147,31 +154,79 @@ def exercises_page():
     )
 
 
+def _create_comment(session_id: int, solution: Solution):
+    user = User.get_or_none(User.id == session_id)
```

Comment:
Receive the argument with the name `user_id`

Reply:
but this actually the row from the db. not the id